bazel/thirdparty/go.work

```diff
@@ -0,0 +1,10 @@
+// This should match what is listed in MODULE.bazel
+go 1.23.0
+
+// Everything we build with Bazel goes here.
```

Comment:
I guess I need to learn what a go workspace is?

Reply:
No you don't, just think of this file as something that connects all the different `go.mod` files in the repo for Bazel to slurp up and compute the third party deps for.

Reply:
yum